### PR TITLE
ee9 update for wsscxf.saml fat

### PR DESCRIPTION
**Pull request by jimmy1wu**
for https://github.com/OpenLiberty/open-liberty/issues/17690

### CODE REVIEW

**Comment by jimmy1wu** [comment]
#build

**Comment by jimmy1wu** [comment]
#libby
#build

**Comment by jimmy1wu** [comment]
#build

**Comment by jimmy1wu** [comment]
#libby

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.saml/fat/src/com/ibm/ws/wssecurity/fat/cxf/samltoken/OneServerTests/CxfSAMLBasic1ServerTests.java [hunk 1]
```diff
@@ -207,9 +205,8 @@ public void CxfSAMLBasicTests_wantAssertionsSignedTrue_missingSignatureEE7Only()
         genericSAML(_testName, webClient, updatedTestSettings, standardFlow, expectations);
     }
 
-    //3/2021 to run with EE8, then the corresponding error message can be expected
-    @SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
-    @AllowedFFDC(value = { "java.util.MissingResourceException", "org.apache.wss4j.common.ext.WSSecurityException" }) //@AV999
+    @SkipForRepeat({ NO_MODIFICATION })
+    @AllowedFFDC(value = { "org.apache.wss4j.common.ext.WSSecurityException" }, repeatAction = { EE8FeatureReplacementAction.ID })
```
Comment (meiaus) [inline dev/com.ibm.ws.wssecurity_fat.wsscxf.saml/fat/src/com/ibm/ws/wssecurity/fat/cxf/samltoken/OneServerTests/CxfSAMLBasic1ServerTests.java:209]: To confirm, `java.util.MissingResourceException` is not needed here?

**Comment by jimmy1wu** [comment]
#build

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.saml/fat/src/com/ibm/ws/wssecurity/fat/cxf/samltoken/OneServerTests/CxfSAMLBasic1ServerTests.java (hunk 1 above)
Comment (jimmy1wu) [reply dev/com.ibm.ws.wssecurity_fat.wsscxf.saml/fat/src/com/ibm/ws/wssecurity/fat/cxf/samltoken/OneServerTests/CxfSAMLBasic1ServerTests.java:209]: right, it is not needed. the test did not throw that particular ffdc

Comment (meiaus) [reply dev/com.ibm.ws.wssecurity_fat.wsscxf.saml/fat/src/com/ibm/ws/wssecurity/fat/cxf/samltoken/OneServerTests/CxfSAMLBasic1ServerTests.java:209]: ok thanks for confirmation

**Review by meiaus — APPROVED** [review]

**Review by arunavemulapalli — APPROVED** [review]

**Comment by arunavemulapalli** [comment]
for #12449

**Pull request merged**